src/status_im/utils/utils.cljs

```diff
@@ -23,6 +23,8 @@
   ([title content confirm-button-text on-accept]
    (show-confirmation title content confirm-button-text on-accept nil))
   ([title content confirm-button-text on-accept on-cancel]
+   (show-confirmation title content confirm-button-text on-accept on-cancel "destructive"))
+  ([title content confirm-button-text on-accept on-cancel ios-confirm-style]
```

Comment:
Can you make the last argument simply a key of a map? To make sure we stop adding extra parameters :)

Reply:
Done!